Dockerfile.tmpl

```diff
@@ -122,21 +119,20 @@ RUN pip install spacy && \
 {{ end}}
 
 # Install PyTorch
+# b/356397043: magma-cuda121 is the latest version
 {{ if eq .Accelerator "gpu" }}
 COPY --from=torch_whl /tmp/whl/*.whl /tmp/torch/
-RUN mamba install -y -c pytorch magma-cuda${CUDA_MAJOR_VERSION}${CUDA_MINOR_VERSION} && \
```

Comment:
I think we should comment here that we're on 12.3 but that magma-cuda123 doesn't exist?

Reply:
ack , in a follow up